dynamiq/nodes/tools/basetool.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+from typing import Any, Literal
+
+from pydantic import BaseModel, Field, ValidationInfo, field_validator
+
+from dynamiq.nodes import NodeGroup
+from dynamiq.runnables import RunnableConfig
+
+
+class DefaultInputSchema(BaseModel, ABC):
+    input: str
+
+
+class ToolMixin:
+    group: Literal[NodeGroup.TOOLS] = NodeGroup.TOOLS
+    name: str = "base_tool"
+    description: str = "base tool"
+    input_schema: type[BaseModel] = Field(default=DefaultInputSchema)
```

Comment:
Please use class variable, not intance variable. So schema available on class level too
```suggestion
    input_schema: ClassVar[type[BaseModel]] = DefaultInputSchema
```